volumes/miovision/sql/function/function-clear-volumes_15min_mvt.sql

```diff
@@ -0,0 +1,66 @@
+CREATE OR REPLACE FUNCTION miovision_api.clear_15_min_mvt(
+    start_date timestamp,
+    end_date timestamp,
+    intersections integer [] DEFAULT ARRAY[]::integer []
+)
+RETURNS void
+LANGUAGE 'plpgsql'
+
+COST 100
+VOLATILE
+AS $BODY$
+
+DECLARE
+    target_intersections integer [] =
+        CASE WHEN CARDINALITY(intersections) = 0
+            --switch out a blank array for all intersections
+            THEN (SELECT ARRAY_AGG(intersections.intersection_uid) FROM miovision_api.intersections)
+            ELSE intersections
+        END;
+    n_deleted integer;
+
+BEGIN
+
+    WITH aggregate_delete AS (
+        DELETE FROM miovision_api.volumes_15min_mvt
+        WHERE
+            intersection_uid = intersection
```

Comment:
should this be `= ANY(target_intersections)` similar to clear_volumes?

Reply:
Good catch